src/rest-server/src/templates/yarnContainerScript.mustache

```diff
@@ -179,17 +181,19 @@ get_host_list $ENV_LIST
 # Backward compatibility
 cat <<ENV >> $ENV_LIST
 # Backward compatibility
+PAI_CURRENT_CONTAINER_IP=$CONTAINER_IP
+PAI_TASK_ROLE_INDEX=$TASK_INDEX
+ENV
+cat <<'ENV' >> $ENV_LIST
 PAI_USERNAME={{{ jobData.userName }}}
```

Comment:
Any reasons for print ENVs and add to ENV_LIST file multiple times?

Reply:
The different in two append is one is `cat <<ENV >> $ENV_LIST` and the other one is `cat <<'ENV' >> $ENV_LIST`.

The first heredoc will substitute all `$VARIABLE`s with their values, as the second one will not, so if we inject `$PATH` to `taskData.name`, it will be save.